react/features/connection-indicator/components/web/ConnectionIndicator.js

```diff
@@ -283,7 +286,10 @@ class ConnectionIndicator extends AbstractConnectionIndicator<Props, State> {
             === JitsiParticipantConnectionStatus.INACTIVE) {
             return (
                 <span className = 'connection_ninja'>
-                    <i className = 'icon-ninja' />
+                    <Icon
+                        className = 'icon-ninja'
+                        size = '1.5em'
```

Comment:
Is there a reason why this one is larger? Perhaps we can have a const if groups of icons have different sizes?